upbound_uxp/assets/dashboards/upbound_uxp_min_set.json

```diff
@@ -0,0 +1,614 @@
+{
+	"title": "Upbound UXP Min Set",
+	"description": "## Upbound UXP Min Set\n\nThis dashboard covers graphs for the UXP Min Set metrics. They are as follows\n\n# Upjet provider Terraform CLI invocations\nupjet_terraform_active_cli_invocations\n\n# Upjet provider Terraform running processes\nupjet_terraform_running_processes\n\n# Total number of reconciliations per controller\ncontroller_runtime_reconcile_total\n\n# Number of goroutines that currently exist\ngo_goroutines\n\n# Number of bytes allocated and still in use\ngo_memstats_alloc_bytes\n\n# Number of bytes allocated and still in use\ngo_memstats_alloc_bytes_total\n\n# Total user and system CPU time spent in seconds\nprocess_cpu_seconds_total\n\n# Resident memory size in bytes\nprocess_resident_memory_bytes\n\n# Start time of the process since unix epoch in seconds\nprocess_start_time_seconds\n\n# Number of HTTP requests partitioned by status code, method, and host.\nrest_client_requests_total\n\n# Total number of adds handled by workqueue\nworkqueue_adds_total\n\n# Current depth of workqueue\nworkqueue_depth\n\n# How long in seconds processing an item from workqueue takes\nworkqueue_work_duration_seconds_bucket\nworkqueue_work_duration_seconds_count\nworkqueue_work_duration_seconds_sum\n\n",
```

Comment:
Can you please remove the headers (anything wrapped in '#') from this dashboard description and instead provide the metrics as a list?

Also, can you please add the Upbound logo to this dashboard?

Reply:
I was exploring how to add logos to dashboards, but did not see a way to add those. Please advise perhaps with a dashboard edit screenshot where to add it. Thanks.